core/src/main/scala/org/apache/spark/ui/env/EnvironmentPage.scala

```diff
@@ -26,11 +26,17 @@ import org.apache.spark.ui.{UIUtils, WebUIPage}
 private[ui] class EnvironmentPage(parent: EnvironmentTab) extends WebUIPage("") {
   private val listener = parent.listener
 
+  private def removePass(kv: (String, String)): (String, String) = {
+    if (kv._1.toLowerCase.contains("password")) {
+      return (kv._1, "******")
+    }
+    kv
+  }
   def render(request: HttpServletRequest): Seq[Node] = {
     val runtimeInformationTable = UIUtils.listingTable(
       propertyHeader, jvmRow, listener.jvmInformation, fixedWidth = true)
     val sparkPropertiesTable = UIUtils.listingTable(
-      propertyHeader, propertyRow, listener.sparkProperties, fixedWidth = true)
+      propertyHeader, propertyRow, listener.sparkProperties.map(removePass), fixedWidth = true)
```

Comment:
This may be just taste, but isn't this easier inline?

```
val redactedProperties = listener.sparkProperties.map { case (key, value) => 
  (key, if (key.toLowerCase.contains("password")) "******" else value)
}
```

Are there any other instances where properties are displayed that need redacting?